compiler/ballerina-lang/src/main/resources/compiler.properties

```
@@ -689,6 +689,9 @@ error.incompatible.iterator.function.signature=\
 error.iterable.not.supported.operation=\
   operation ''{0}'' does not support given collection type
 
+error.empty.closed.record.cannot.be.iterated=\
+  empty closed record cannot be iterated
```

```suggestion
  cannot iterate a closed record with no fields
```

Please update the code and key too, if changing.